test/backend_api.cpp

```diff
@@ -25,9 +25,9 @@ using namespace ngraph;
 TEST(backend_api, registered_devices)
 {
     vector<string> devices = runtime::Backend::get_registered_devices();
-    EXPECT_GE(devices.size(), 1);
+    EXPECT_GE(devices.size(), 0);
 
-    EXPECT_TRUE(contains(devices, "INTERPRETER"));
+    // EXPECT_TRUE(contains(devices, "INTERPRETER"));
```

Comment:
Are you planning on enabling this in the future?  If not, it's best to go ahead and remove it.

Reply:
Yes. I want to re-enable this feature. It should be fairly easy but is not needed for this PR.